registration_form/registration_form.js

```diff
@@ -0,0 +1,48 @@
+function validUname(uname) {
+    let error = 0;
+    var length = uname.length;
+    var validChars = /\w{3,10}/;
+    document.getElementById('username_error').innerHTML = '';
+    if (length < 3 || length > 10 || !validChars.test(uname)) {
+        error++;
+        document.getElementById('username_error').innerHTML = "<em style='color:red'>Please enter valid username</em>";
+    }
+    return error;
+}
+function validPass(pass) {
+    let error = 0;
+    var length = pass.length;
+    var validChars = /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])[0-9a-zA-Z]{6,}$/;
```

Comment:
http://webtech.w3c.fmi.uni-sofia.bg/w14labs/regex.html#2 "Now they have two problems."
Would you say that this code is readable/easily understandable? Or easily modifiable by people who do not know regex? Also how error-prone is it to bugs?
Idea to consider: using multiple and simpler regex-es and applying "and" condition if all of them match